BearlyHumans/squirrelville
# Human squash

This PR request adds to features regarding humans
* Small change to how friendly humans give's acorns
* Human now has an ending behavior when player is in giant mode

Steps to test Human-Friendly change
* be in lachie scene
* Set human to "lying" idle animation
* Set human to "friendly" 
* Run game and walk up to human
* Watch for animation 

https://user-images.githubusercontent.com/20939519/140290263-e54cd182-f719-48cf-98fb-155f98a954b4.mp4

Steps to test Human ending:
* Be in lachie scene
* Eat giant acorn 
* Roll towards player
* check if player runs away from player
* roll into player to test animation 
* spit out giant acorn and see if human lies back down

https://user-images.githubusercontent.com/20939519/140290630-8c898a72-2d42-4a81-82de-110dd999089b.mp4
